examples/howto/js_events.py

```diff
@@ -99,5 +99,7 @@ def display_event(div, attributes=[]):
 # Selection events
 p.js_on_event(events.SelectionGeometry, display_event(div, attributes=['geometry', 'final']))
 
+curdoc().on_event("document_ready", display_event(div))
```

Comment:
This doesn't work currently, because `show()`, `save()`, etc. disregard the document.

Reply:
I think there was a similar comment maybe on discourse about setting title on curdoc not working for standalone